test-app/tests/unit/dom/click-test.js

```diff
@@ -202,6 +203,17 @@ module('DOM Helper: click', function (hooks) {
   module('focusable element types', function () {
     let clickSteps = ['mousedown', 'focus', 'focusin', 'mouseup', 'click'];
 
+    if (isChrome) {
+      clickSteps = [
+        'mousedown',
+        'focus',
+        'focusin',
+        'mouseup',
+        'click',
+        'selectionchange',
```

Comment:
Should we just push `selectionchange` to the existing list?

Reply:
then we'd need to add inverted conditions for safari and firefox